version.txt

```diff
@@ -1 +1 @@
-11.0.24.20.8-GA
\ No newline at end of file
+11.0.25.21.9-GA
```

Comment:
diff --git a/version.txt b/version.txt
index 48d075b2d5..b2d0310003 100644
--- a/version.txt
+++ b/version.txt
@@ -1 +1 @@
-11.0.25.21.9-GA
\ No newline at end of file
+11.0.24.20.8-GA
\ No newline at end of file